docs/General/install.rst

```diff
@@ -8,8 +8,23 @@ Freeciv21 has a number of prerequisites.  Note, that apart from the first prereq
 configuration process is smart enough to work out whether your system is suitable. If in doubt, just try it.
 
 An operating system that support Qt
-    Any modern operating system that support Qt is required. As of this writing this is Linux, Microsoft
-    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows Msys2 is supported.
+    Any modern operating system that supports Qt 5.12+ is required. As of this writing this is Linux, Microsoft
+    Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows MSYS2 (MingW) is supported.
+
+    Linux Distributions:
+
+    * Arch Current

```

Comment:
What does arch current mean ? Is that a term ? I use arch and afaik, people just arch to denote it. Because it is rolling release.

Reply:
I think it means that you are keeping it up to date aka current.

Reply:
Well, isnt that assumed? All these are supported only after they keep it current. We cant and the users should not expect us to support outdated distro even if it is on the list. Maybe add a line that everything should be current and skip the current here ?